app/content/tiles/palettes.ts

```diff
@@ -201,7 +201,51 @@ paletteHash.pits = [
     [null, 1846, 1,    null, null],
 ];
 
+paletteHash.vanara = [
+    [0, 1, 1847, 0, 1865, 1866, 1867, 0, 0, 0],
```

Comment:
Minor nitpick, but the indentation is strange on the first line of each of these palettes.